apps/web/pages/forgot-password.tsx

```diff
@@ -0,0 +1,30 @@
+import Head from "next/head";
+import { getUserFromToken } from "@documenso/lib/server";
+import ForgotPassword from "../components/forgot-password";
+
+export default function ForgotPasswordPage() {
+  return (
+    <>
+      <Head>
+        <title>Forgot Password | Documenso</title>
+      </Head>
+      <ForgotPassword />
+    </>
+  );
+}
+
+export async function getServerSideProps(context: any) {
```

Comment:
We can just the type provided by Next for this rather than an `any` we could also destructure as well :)

Reply:
To accomplish this, I had to modify and remove all instances of `res` in the getUserToken. `res` from GetServerSidePropsContext was causing type errors with NextApiResponse but  `res.status` is not doing much either since it is in getServerSideProps. 

https://github.com/documenso/documenso/blob/5ec97657c1f1420082aa08f246138a9f212066ce/packages/lib/server/getUserFromToken.ts#L1-L34

https://github.com/documenso/documenso/blob/e3bc41934c69a904c85b054851def2f30e465046/packages/lib/server/getUserFromToken.ts#L1-L27